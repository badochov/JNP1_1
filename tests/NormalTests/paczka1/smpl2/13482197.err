Error in line 2: JBkcdZVmt S53 4,4
Error in line 1: HgrM26 S412 52,0
Error in line 11: GOb9ZHaJuGG S53 81,2
Error in line 19: cQTt S412 8,3
Error in line 31: cQTt S53 72,1
Error in line 13: JBkcdZVmt S412 19,7
Error in line 38: cQTt S66 95,4
