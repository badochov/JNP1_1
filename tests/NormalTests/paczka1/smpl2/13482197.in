HgrM26 S412 52,0
JBkcdZVmt S53 4,4
S412 S66 12,5
JBkcdZVmt S412 30,1
JBkcdZVmt S412 8,7
S412 S66 43,2
S412 S412 94,1
Tgyi S66 44,0

HgrM26 S66 64,8
GOb9ZHaJuGG S53 81,2
?GOb9ZHaJuGG
JBkcdZVmt S412 19,7
Tgyi S66 77,4
GOb9ZHaJuGG S412 53,1
S412 S412 35,8
PcGhirWh S53 26,8

cQTt S412 8,3
cQTt S53 45,6
Tgyi S53 94,7
Tgyi S53 386,4
cQTt S53 87,9
GOb9ZHaJuGG S412 154,0
PcGhirWh S53 126,2
GOb9ZHaJuGG S66 55,2
HgrM26 S66 46,5
?S53
PcGhirWh S53 89,3
PcGhirWh S53 96,4
cQTt S53 72,1
GOb9ZHaJuGG S66 205,3
S412 S412 4,4
?GOb9ZHaJuGG
HgrM26 S66 109,7
S412 S412 18,3

cQTt S66 95,4
JBkcdZVmt A908 58,1
cQTt S53 33,5