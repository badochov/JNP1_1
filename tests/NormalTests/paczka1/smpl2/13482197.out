S53 433,4
GOb9ZHaJuGG S 251,0
